ArcadiaParties.Data/Data/Seed.cs

```diff
@@ -0,0 +1,46 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ArcadiaParties.Data.Models;
+using Newtonsoft.Json;
+
+namespace ArcadiaParties.Data.Data
+{
+    public class Seed : ISeed
+    {
+        private readonly DataContext _context;
+
+        public Seed(DataContext dataContext)
+        {
+            _context = dataContext;
+        }
+
+        public async Task SeedData()
+        {
+            if (_context.Department.Any())
+            {
+                return;
+            }
+
+            var departmentData = System.IO.File.ReadAllText("../ArcadiaParties.Data/Data/Depart.json");
+            var departments = JsonConvert.DeserializeObject<List<Department>>(departmentData);
+            await _context.Department.AddRangeAsync(departments);
+            await _context.SaveChangesAsync();
```

Comment:
Also, you don't need to call `SaveChangesAsync` each time, you can call it once in the end of database update.